tgc-notes-fe/src/pages/HomePage.tsx

```diff
@@ -18,11 +23,48 @@ const NOTES = gql`
   }
 `;
 
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    box: {
+      height: "100vh",
+      display: "flex"
+    },
+    noteList: {
+      maxWidth: "30%"
+    },
+    noteDetail: {
+      width: "65%"
+    }
+  }),
+);
+
+
 const HomePage = (props: RouteComponentProps) => {
+
+  const classes = useStyles();
+
   const { loading, error, data } = useQuery<NotesReponse>(NOTES);
+  const [displayedNote, setDisplayedNote] = useState(data?.notes[0]);
+
+  useEffect(() => {
+    if (data?.notes) {
+      const targetNote = data.notes.find(note => displayedNote?.id === note.id);
 
-  console.log('Loading in home page', loading)
-  console.log('Notes in home page', data?.notes)
+      setDisplayedNote(targetNote)
+
+      // select first note on first load
+      if (displayedNote == undefined) {
+        setDisplayedNote(data?.notes[0])
+      }
+    }
+  }, [data?.notes])
+
+  useEffect(() => {
+    // after deleting a note, the first is picked
```

Comment:
Maybe change this in way that we display next or previous note instead?

Reply:
I changed it. However, please review, if it looks fine or you have any suggestions. Browser is also giving me some warnings for the useEffect fields. About missing dependencies. Not sure what that means (it works though).